.circleci/config.yml

```diff
@@ -91,7 +91,7 @@ commands:
             . /opt/conda/etc/profile.d/conda.sh
             mkdir /logs
             mamba --version
-            mamba install -c conda-forge "mamba>=2"  # mamba<2 can't solve env
+            mamba install -c conda-forge "mamba>=2,<2.0.8"  # mamba<2 can't solve env
```

Comment:
Do we need this now?

Reply:
No we don't, I was trying to see why the heck conda can't install mamba

Reply:
getting to exasperation here, man - it appears mamba==2.0.7 suffers from a bug, mamba<2 is absolutely impotent, and Circle keeps complaining it takes too long to install mamba>2 when two days ago everything was a-ok. Maybe we should use `mambaorg/micromamba:latest` after all - am gonna plug that in if this last test fails. BTW the RTD build is now superfast, thanks to your idea! :beer: